appveyor.yml

```diff
@@ -149,11 +163,14 @@ install:
 
   # Replace numpy distutils with a version that can build with msvc + mingw-gfortran.
   - ps: |
+      $PYTHON_VERSION = $env:PYTHON_VERSION
+      If ($PYTHON_VERSION -ne 3.8) {
```

Comment:
In the most recent commit I've added this guard to see if blocking the distutils override/patching can allow Appveyor to at least build SciPy with Python 3.8 on Windows (32 & 64-bit).

This has been in various parts of our CI for quite some time because of a long-standing distutils limitation for the lower versions we supported I think.

It would probably be more precise to put the guard on the version of NumPy, but this should suffice for testing at least.